packages/cfd/build/constants.js

```diff
@@ -0,0 +1,152 @@
+    Constants: path.resolve(__dirname, '../src/Constants'),
+    Components: path.resolve(__dirname, '../src/Components'),
+    Containers: path.resolve(__dirname, '../src/Containers'),
+    Modules: path.resolve(__dirname, '../src/Modules'),
+    Sass: path.resolve(__dirname, '../src/sass'),
+    Stores: path.resolve(__dirname, '../src/Stores'),
+};
+
+const rules = (is_test_env = false, is_mocha_only = false) => [
+    ...(is_test_env && !is_mocha_only
+        ? [
+              {
+                  test: /\.(js|jsx)$/,
+                  exclude: /node_modules|__tests__|(build\/.*\.js$)|(_common\/lib)/,
+                  include: /src/,
+                  loader: 'eslint-loader',
+                  enforce: 'pre',
+                  options: {
+                      formatter: require('eslint-formatter-pretty'),
+                      configFile: path.resolve(__dirname, '../.eslintrc.js'),
+                      ignorePath: path.resolve(__dirname, '../.eslintignore'),
+                  },
+              },
+              {
+                  test: /\.(ts|tsx)$/,
+                  exclude: /node_modules|__tests__|(build\/.*\.js$)|(_common\/lib)/,
+                  include: /src/,
+                  loader: 'eslint-loader',
+                  enforce: 'pre',
+                  options: {
+                      formatter: require('eslint-formatter-pretty'),
+                      configFile: path.resolve(__dirname, '../.eslintrc.ts'),
+                      ignorePath: path.resolve(__dirname, '../.eslintignore'),
+                  },
+              },
+          ]
+        : []),
+    {
+        test: /\.(js|jsx|ts|tsx)$/,
+        exclude: is_test_env ? /node_modules/ : /node_modules|__tests__/,
+        include: is_test_env ? /__tests__|src/ : /src/,
+        use: js_loaders,
+    },
+    {
+        test: /\.html$/,
+        exclude: /node_modules/,
+        use: html_loaders,
+    },
+    {
+        test: /\.(png|jpg|gif|woff|woff2|eot|ttf|otf)$/,
+        exclude: /node_modules/,
+        use: file_loaders,
+    },
+    {
+        test: /\.svg$/,
+        exclude: /node_modules/,
+        include: /public\//,
+        use: svg_file_loaders,
+    },
+    {
+        test: /\.svg$/,
+        exclude: /node_modules|public\//,
+        use: svg_loaders,
+    },
+    is_test_env
+        ? {
+              test: /\.(sc|sa|c)ss$/,
+              loaders: 'null-loader',
+          }
+        : {
+              test: /\.(sc|sa|c)ss$/,
+              use: css_loaders,
+          },
+];
+
+const MINIMIZERS = !IS_RELEASE
+    ? []
+    : [
+          new TerserPlugin({
+              test: /\.js$/,
+              exclude: /(smartcharts)/,
+              parallel: 2,
+          }),
+          new CssMinimizerPlugin(),
+      ];
+
+const plugins = (base, is_test_env, is_mocha_only) => [
+    new CleanWebpackPlugin(),
+    // new HtmlWebPackPlugin(htmlOutputConfig()),
+    // new HtmlWebpackTagsPlugin(htmlInjectConfig()),
```

Comment:
Are we going to enable these plugins in the future?

Reply:
I don't think so. I remove them in the my new commit.